### PR TITLE
fix(planning): fix planning recall not save on update scheduling

### PR DESCRIPTION
**Pull request by stonebuzz**
When opening a task already created the field 'begin' is reset to null by the function getFromDB.

It's no longer possible to create a schedule reminder after creating a task.

This PR prevent this

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #6250

### CODE REVIEW

File: tests/functionnal/TicketTask.php [hunk 1]
```diff
@@ -57,6 +57,109 @@ private function getNewTicket() {
...
+      //create one task with scheduling and recall
+      $task = new \TicketTask();
+      $this->integer(
+         $task->add([
+            'state'           => \Planning::TODO,
+            'tickets_id'      => $ticketId,
+            'actiontime'      => "3600", //1hours
+            'description'     => "Task with schedul and recall",
+            'users_id_tech'   => $uid,
+            'plan'           => ['begin' => $date_begin,
```
Comment (trasher) [inline tests/functionnal/TicketTask.php]: s/$date_begin/$formatted/

File: tests/functionnal/TicketTask.php [hunk 2]
```diff
@@ -57,6 +57,112 @@ private function getNewTicket() {
...
+   public function testSchedulingAndRecall() {
+      $this->login();
+      $ticketId = $this->getNewTicket();
+      $uid = getItemByTypeName('User', TU_USER, true);
+
+      $date_begin = new \DateTime(); // ==> now
+      $date_begin_string = $date_begin->format('Y-m-d H:i:s');
+
+      $date_end = new \DateTime(); // ==> +2days
+      $date_end->sub(new \DateInterval('P1D'));
```
Comment (trasher) [inline tests/functionnal/TicketTask.php]: s.sub/add/ ;)

Comment (trasher) [reply tests/functionnal/TicketTask.php]: s/P1D/P2D/ also

Comment (stonebuzz) [reply tests/functionnal/TicketTask.php]: fixed

File: tests/functionnal/TicketTask.php [hunk 3]
```diff
@@ -57,6 +57,119 @@ private function getNewTicket() {
...
+                                  '_duraction'   => "172800"],  //period (2 days)
+         '_planningrecall'    => ['before_time' => '14400', //recall 4 hours
+                                  'itemtype'    => 'TicketTask',
+                                  'users_id'    => $uid,
+                                  'field'       => 'begin', //default
+                                 ]
+      ]);
+      $this->integer($task_id)->isGreaterThan(0);
+
+      //load plannig schedul with recall
```
Comment (cedric-anne) [inline tests/functionnal/TicketTask.php]:
```suggestion
      //load planning schedule with recall
```

Comment (stonebuzz) [reply tests/functionnal/TicketTask.php]: fixed

**Pull request merged**